keras_cv/models/task.py

```diff
@@ -142,6 +155,7 @@ def from_preset(
         if metadata["weights_url"].endswith(".weights.h5"):
             local_weights_path = "model.weights.h5"
 
+        # TODO: implement ``get_file`` in ``keras_core``
```

Comment:
This exists as `keras_core.utils.file_utils.get_file`

Reply:
Done, thanks!